### ISSUE TITLE
Update Document on how to pass data from one form to another via contact-summary-templated.js and contact-summary-extars.js

### ISSUE THREAD

**Issue by PrjShrestha** [medic/cht-core#7713]
Update Document on how to pass data from one form to another via contact-summary-templated.js and contact-summary-extars.js. The current existing document could be more newbie friendly. So adding an example of step by step on how to do this would be the end goal.

**Comment by jkuester** [comment]
@PrjShrestha are you talking about expanding [this documentation](https://docs.communityhealthtoolkit.org/apps/reference/contact-page/#care-guides) which discusses (very briefly) how to populate fields in a form with data from the person's contact summary?  I definitely think that would be helpful, but want to be sure I am understanding your request.

**Comment by PrjShrestha** [comment]
@jkuester Yes, I am talking about that documentation. I would like to elaborate on the pregnancy example with a step-by-step example of data being passed in various stages. would be great for people with minimal knowledge of the framework.
